spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/profile/AbstractKubernetesProfileEnvironmentPostProcessor.java

```diff
@@ -41,7 +39,7 @@
  */
 public abstract class AbstractKubernetesProfileEnvironmentPostProcessor implements EnvironmentPostProcessor, Ordered {
 
```

Comment:
the "biggest" change is this. Since this is a `EnvironmentPostProcessor`, `LOG` is not going to work - it is too early for it and it is not set-up (tested locally on a k8s instance). For this purpose there is `DeferredLog`, which I have been using here (tested locally too, of course)